src/main/webapp/definitions/alfresco-content-models.yaml

```diff
@@ -0,0 +1,738 @@
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/QNameEntry'
+          source:
+            $ref: '#/definitions/QName'
+  QNameEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/QName'
+  QName:
+    type: object
+    required:
+      - namespacePrefix
+      - namespaceUri
+      - name
+    properties:
+      namespacePrefix:
+        type: string
+      namespaceUri:
+        type: string
+      name:
+        type: string
+  ModelEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Model'
+  Model:
+    allOf:
+      - $ref: '#/definitions/QName'
+      - properties:
+          version:
+            description: the model version
+            type: string
+          author:
+            description: author of the model
+            type: string
+          description:
+            description: the model description
+            type: string
+  TypeEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Type'
+  Type:
+    allOf:
+      - $ref: '#/definitions/QName'
+      - properties:
+          title:
+            description: the human-readable type title
+            type: string
+          parent:
+            description: parent type defination
+            $ref: '#/definitions/Type'
+          description:
+            description: description of the type
+            type: string
+          archive:
+            description: define if the type is archived on delete
+            type: boolean
+  AspectEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Aspect'
+  Aspect:
+    allOf:
+      - $ref: '#/definitions/QName'
+  ConstraintEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Constraint'
+  Constraint:
+    allOf:
+      - $ref: '#/definitions/QName'
+      - properties:
+          constraint:
+            description: implementation details about the constrait
+            type: object
+            additionalProperties:
+              type: object
+  PropertyPaging:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/PropertyEntry'
+          source:
+            $ref: '#/definitions/Property'
+  PropertyEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Property'
+  Property:
+    type: object
+    required:
+      - mandatory
+    allOf:
+      - $ref: '#/definitions/QName'
```

Comment:
A property definition has more than just QName attributes see: https://github.com/Alfresco/alfresco-data-model/blob/c7aaa53cce8245889ef79daffac46b53f827ee0e/src/main/java/org/alfresco/service/cmr/dictionary/PropertyDefinition.java

Reply:
You are right, we miss a lot of attributes, I think that we should add the attributes below:
- Title
- Description
- DefaultValue
- DataType (QName)
- Class (QName)
- Override
- MultiValued
- Mandatory
- MandatoryEnforced
- Protected
- Indexed